test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,93 @@
+package seedu.addressbook.commands;
+
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.util.TestUtil;
+
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+public class SortCommandTest
+{
+    private static final List<ReadOnlyPerson> EMPTY_PERSON_LIST = Collections.emptyList();
+    private static final Set<String> EMPTY_STRING_LIST = Collections.emptySet();
+
+    @Test
+    public void sortCommand_book_is_sorted_2_persons() {
```

Comment:
`sortCommand_book_is_sorted_2_persons` -> `sortCommand_book_isSorted` 

The number of persons is up to the tester. If 2 is enough to prove that its working, then its fine. No need to include them into method name. 

`is` and `sorted` can be combined otherwise it will be too fine grained